config/locales/pt-BR.yml

```diff
@@ -23,6 +23,7 @@ pt-BR:
       updated_notice: Fórum de mensagem foi atualizado
     messageboard_group:
       create: Criar um novo grupo de mensagens
+      saved: Groupo de mensagens %{name} salvou
```

Comment:
Man, no apologies necessary. We're lucky you even know this exists so any and all help is greatly appreciated

Thank you, Ramiro :)